<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputHexagramDlg</class>
 <widget class="QDialog" name="InputHexagramDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InputHexagramDlg</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>排盘</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>420</width>
              <height>350</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
            <property name="pixmap">
             <pixmap resource="QDivination.qrc">:/QDivination/image/guize.jpg</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>阳历：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateTimeEdit" name="dateTimeEdit"/>
              </item>
              <item>
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>性别：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cbSex">
                <item>
                 <property name="text">
                  <string>男</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>女</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QGridLayout" name="gridLayout">
                <property name="verticalSpacing">
                 <number>15</number>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_11">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>六爻：</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="cb_6">
                  <item>
                   <property name="text">
                    <string>少阳-</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>少阴- -</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>老阳O</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>老阴X</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_10">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>五爻：</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QComboBox" name="cb_5">
                  <item>
                   <property name="text">
                    <string>少阳-</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>少阴- -</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>老阳O</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>老阴X</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>四爻：</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QComboBox" name="cb_4">
                  <item>
                   <property name="text">
                    <string>少阳-</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>少阴- -</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>老阳O</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>老阴X</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>三爻：</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QComboBox" name="cb_3">
                  <item>
                   <property name="text">
                    <string>少阳-</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>少阴- -</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>老阳O</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>老阴X</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>二爻：</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QComboBox" name="cb_2">
                  <item>
                   <property name="text">
                    <string>少阳-</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>少阴- -</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>老阳O</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>老阴X</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>初爻：</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="QComboBox" name="cb_1">
                  <item>
                   <property name="text">
                    <string>少阳-</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>少阴- -</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>老阳O</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>老阴X</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QPushButton" name="btnOk">
              <property name="text">
               <string>排盘</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnCancel">
              <property name="text">
               <string>取消</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QDivination.qrc"/>
 </resources>
 <connections/>
</ui>
